LICENSE

```diff
@@ -0,0 +1,16 @@
+Copyright (C) 2016 CERN.
```

Comment:
Could you please move this license to ``.license`` and add this [one](https://github.com/inveniosoftware/invenio-search-js/blob/master/LICENSE) to this file? :)